dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -32,10 +33,12 @@ const createValidationSchema = (key: string, formSchema: FormPageSchema) => {
     case "number":
       validationShape = number();
       break;
+    case "array":
+      validationShape = array().min(1, REQUIRED_MESSAGE).of(string());
```

Comment:
Moving this into the if statement on line 40 doesn't work - only validates the first time, you can go back and unselect the boxes and it will allow you through